### PR TITLE
feat(banxa): support USDC and USDT sell orders

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

This adds support for the newly supported USDC and USDT stablecoins for Banxa sell orders.

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

## Risk

None, this is just adding some new supported sell assets, for Banxa only

## Testing

- Have 50$+ of USDC/USDT in your wallet, or monkey-patch this line to `disabled={false}`:
https://github.com/shapeshift/web/blob/8770737be7fe61363970659bfaf51d76c1361377/src/components/Modals/FiatRamps/views/Overview.tsx#L250
- Go to Banxa ramp option's sell tab
- USDC and USDT should be available as sell assets
- Banxa partner tab should open with USDC/USDT populated

## Screenshots (if applicable)

image
image
image

### CODE REVIEW

**Review by reallybeard — DISMISSED** [review]

**Review by elmutt — APPROVED** [review]

File: src/components/Modals/FiatRamps/config.ts [hunk 1]
```diff
@@ -82,7 +85,9 @@ export const supportedFiatRamps: SupportedFiatRamp = {
        * https://discord.com/channels/554694662431178782/972197500305948803/973110904382169118
        * banxa only supports btc sells for now
```
Comment (0xdef1cafe) [inline src/components/Modals/FiatRamps/config.ts:86]: delete comment?

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]
seems like this can be simplified?

File: src/components/Modals/FiatRamps/config.ts [hunk 2]
```diff
@@ -82,7 +85,9 @@ export const supportedFiatRamps: SupportedFiatRamp = {
...
        */
-      const sellAssets = buyAssets.filter(a => a.assetId === btcAssetId)
+      const sellAssets = buyAssets.filter(a =>
+        [btcAssetId, usdcAssetId, usdtAssetId].includes(a.assetId),
+      )
```
[the remark this answers is not in the document]
Comment (gomesalexandre) [reply src/components/Modals/FiatRamps/config.ts:90]: That wouldn't work out, we need this to be a `FiatRampAsset[]`, not an `AssetId[]` - we would also lose the safety of `getSupportedBanxaAssets`

File: src/components/Modals/FiatRamps/config.ts (hunk 1 above)
Comment (gomesalexandre) [reply src/components/Modals/FiatRamps/config.ts]: Done in 62c6451

**Review by 0xdef1cafe — APPROVED** [review]

**Comment by 0xdef1cafe** [comment]
merging as previously approved and only removed comment

**Pull request merged**